projects/rero/ng-core/src/lib/menu/menu-item.ts

```diff
@@ -138,7 +138,7 @@ export class MenuItem implements ItemInterface {
    * @return boolean
    */
   hasRouterLink(): boolean {
-    return isArray(this._routerLink) && this._routerLink.length > 0
+    return Array.isArray(this._routerLink) && this._routerLink.length > 0
     ? true
     : false;
```

Comment:
Note : We could simplify with only `return (Array.isArray(this._routerLink) && this._routerLink.length > 0);`